File: Hire-Quest(Backend)/java/com/alok91340/gethired/dto/UserDto.java
/**
 * 
 */
package com.alok91340.gethired.dto;

import lombok.Data;

/**
 * @author alok91340
 *
 */
@Data
public class UserDto {
	private Long id;
	private String username;
	private String name;
	private String email;
	private String headline;
	private String birthdate;
	private String currentOccupation;
	private boolean status;
	private String phone;
	private int isRecuriter;
	private String gender;
}

File: HireQuest(Frontend)/app/src/main/java/com/example/gethired/entities/UserDto.kt
package com.example.gethired.entities

import java.io.Serializable

data class UserDto(
    val id:Long,
    val birthdate: String?,
    var currentOccupation: String?,
    val email: String,
    var headline: String?,
    var name: String,
    val phone: String?,
    var status: Boolean?,
    var username: String,
    var isRecuriter:Int?,
    val gender:String?

) : Serializable